lib/aur-build

```diff
@@ -201,6 +202,14 @@ if [[ -v results_file ]]; then
     : >"$results_file" # truncate file
 fi
 
+if [[ -v makepkg_user ]]; then
+    id -- "$makepkg_user" >/dev/null
+
+    makepkg() {
+        command sudo -u "$makepkg_user" makepkg "$@"
```

Comment:
I don't know if it's possible that this line becomes recursive, but if you are not sure either, maybe it would be safer to resolve the full path to `makepkg`, or just hardcode `/usr/bin/` ?

Reply:
It's fine because `makepkg()` is a function not exported to `sudo`. Even the `command` here is not required but I added it for clarity.